ci_environment/docker/recipes/default.rb

```diff
@@ -18,10 +18,15 @@
   action :add
 end
 
-docker_pkg = 'docker-engine'
-# TODO docker_pkg += "-#{node['docker']['version']}" if node['docker']['version']
-
-package %W(linux-image-extra-#{`uname -r`.chomp} lxc #{docker_pkg})
+package %W(
+  linux-image-extra-#{`uname -r`.chomp}
```

Comment:
This gets installed at packer time, so there'll always a reboot prior to use.  Do you mean the `*-image-extra*` package is unnecessary also because of the vivid kernel installation?